src/sentry/web/urls.py

```diff
@@ -593,11 +599,6 @@ def init_all_applications():
         react_page_view,
         name='sentry-stream'
     ),
-    url(
-        r'^(?P<organization_slug>[\w_-]+)/(?P<project_slug>[\w_-]+)/(?:group|issues)/(?P<group_id>\d+)/events/(?P<event_id_or_latest>(\d+|latest))/json/$',
-        GroupEventJsonView.as_view(),
-        name='sentry-group-event-json'
-    ),
```

Comment:
Do we want to keep the old URL around for a day or so until folks reload the JS app? Right now those folks would get a 404.

Reply:
That does makes sense but I'm gonna punt on this link being non critical and just do it in one hit